client/src/Components/ProfileDropdown/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { AiTwotoneEdit } from 'react-icons/ai';
+import { IoLogOutOutline } from 'react-icons/io5';
+
+import './profiledropdn.scss';
+
+const ProfileDropdown = ({ handleEdit, handleExit }) => {
+  return (
+    <>
+      <section className="profDropDn">
+        <div className="profDropDn__edit" onClick={handleEdit}>
+          <AiTwotoneEdit style={{ height: '4em', width: '2em', paddingLeft: '1rem' }} />
+          <p className="profDropDn__edit profDropDn__edit__text">Profile</p>
+        </div>
+
+        <div className="profDropDn__logout" onClick={handleExit}>
+          <IoLogOutOutline style={{ height: '4em', width: '2em', paddingLeft: '1rem' }} />
+          <p className="profDropDn__logout profDropDn__logout__text">LogOut</p>
+        </div>
+      </section>
```

Comment:
"profDropDn__edit profDropDn__edit__text"  i didn't understand this one. Here you have two elements under profDropDn__edit. as we cannot go further than 1 step the best way to name class for p and the icon is profDropDn__text and profDropDn__icon. 

Also please give classNames to the icons and give styling from the SCSS. Maybe we cannot give color from the SCSS file. not sure try. if it doesn't work then give color with inline styles.